audioipc/src/rpccore.rs

```diff
@@ -137,8 +202,8 @@ impl<C: Client> Handler for ClientHandler<C> {
 
     fn consume(&mut self, response: Self::In) -> Result<()> {
         trace!("ClientHandler::consume");
-        if let Some(complete) = self.in_flight.pop_front() {
-            drop(complete.send(response));
+        if let Some(proxy) = self.in_flight.pop_front() {
+            self.proxies.send(proxy, response);
```

Comment:
nit: Add a comment explaining `response` is paired to `proxy` since it's a synchronous returned value from `proxy`-paired `Request`?